src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -2936,12 +2936,11 @@ namespace HeatBalanceAirManager {
 				} else if ( SELECT_CASE_var == "AIRFLOWNETWORK" ) {
 					AirModel( ZoneNum ).AirModelType = RoomAirModel_AirflowNetwork;
 					AirModel( ZoneNum ).SimAirModel = true;
-					ValidateComponent( "RoomAirSettings:AirflowNetwork", cAlphaArgs( 2 ), IsNotOK, "GetRoomAirModelParameters" );
-					if ( IsNotOK ) {
-						ShowContinueError( "In " + cCurrentModuleObject + '=' + cAlphaArgs( 1 ) + '.' );
+					if ( GetNumObjectsFound( "AirflowNetwork:SimulationControl" ) == 0 ) {
+						ShowSevereError( "In " + cCurrentModuleObject + " = " + cAlphaArgs( 1 ) + ": " + cAlphaFieldNames( 3 ) + " = AIRFLOWNETWORK."  );
+						ShowContinueError( "The AirflowNetwork model inputs are required." );
```

Comment:
@mjwitte I like your suggestion. The RoomAirflowNetwork model requires a whole set of AirflowNetwork inputs. I am going to make changes.